### DISCUSSION TITLE
ERROR Unable to prepare the list of endpoints: strconv.ParseFloat: parsing "": invalid syntax

### DISCUSSION THREAD

**Discussion by pasztorl** [minio/minio#20438] [Q&A] [answered]
Hi,

I got this message when upgrading from RELEASE.2024-05-28T17-19-04Z to RELEASE.2024-06-04T19-20-08Z
I've tried latest release too with the same result.
I've searched the source code and can't find this error message. How can I debug it?

**Comment by harshavardhana** [comment]
Share the server configuration..

**Comment by huguesgr** [comment]
I had the same issue when upgrading from RELEASE.2024-07-31T05-46-26Z to latest. Removing `MINIO_MEMLIMIT` from my configuration fixed the issue.

**Comment by pasztorl** [comment]
interesting, tried to remove this and the problem still exists

**Comment by pasztorl** [comment]
config,env:
```
export GOMEMLIMIT="4GiB"
export MINIO_ARGS="https://minio-pool-0-{0...2}.minio-hl.devel.svc.k8s.example.com/export{0...1}"
export MINIO_BROWSER="on"
export MINIO_BROWSER_REDIRECT_URL="https://console.minio.devel.example.com"
export MINIO_DOMAIN="minio.devel.example.com"
export MINIO_IDENTITY_OPENID_CLAIM_NAME="groups"
export MINIO_IDENTITY_OPENID_CLIENT_ID="example.com"
export MINIO_IDENTITY_OPENID_CLIENT_SECRET="xxx"
export MINIO_IDENTITY_OPENID_COMMENT="yyy"
export MINIO_IDENTITY_OPENID_CONFIG_URL="https://sso.example.comt/realms/xxx/.well-known/openid-configuration"
export MINIO_IDENTITY_OPENID_REDIRECT_URI="https://console.minio.devel.example.com/oauth_callback"
export MINIO_IDENTITY_OPENID_SCOPES="profile email"
export MINIO_MEMLIMIT="4GiB"
export MINIO_PROMETHEUS_JOB_ID="minio-job"
export MINIO_ROOT_PASSWORD="vvv"
export MINIO_ROOT_USER="nnn"
export MINIO_SERVER_URL="https://minio.devel.example.com"
export MINIO_STORAGE_CLASS_STANDARD="EC:1"
export MINIO_UPDATE="on"
export MINIO_UPDATE_MINISIGN_PUBKEY="zzz"
```

**Comment by pasztorl** [comment]
sorry, yes, if i remove MINIO_MEMLIMIT it works, but how to limit minio memory? can't find in the docs..

**Comment by pasztorl** [comment, marked as answer]
so.. MINIO_MEMLIMIT set to memlimit variable which is uint64 so you need to define in bytes

**Comment by LuxTheDude** [comment]
Setting MINIO_MEMLIMIT as a unit64 still results in the error above for me. Removing MINIO_MEMLIMIT removes the error. However i would still like to set a memory limit.
